modules/configuration/README.md

```diff
@@ -14,6 +14,79 @@ internal implementations to avoid writing boilerplate code.
 
 All schema classes must end with the `ConfigurationSchema` suffix.
 
+### Polymorphic configuration
```

Comment:
Actually, when working with this type of configuration I found out that you also need to explicitly pass a list of all polymorphic extensions to the `ConfigurationRegistry`. This is not documented anywhere, as far as I know

Reply:
Fixed, I guess